spec/models/bulkrax/csv_entry_spec.rb

```diff
@@ -121,21 +121,97 @@ module Bulkrax
           allow(subject).to receive(:record).and_return(
             'source_identifier' => '2',
             'title' => 'some title',
-            'creator_first_name' => 'Fake',
-            'creator_last_name' => 'Fakerson',
-            'creator_position' => 'Leader, Jester, Queen',
-            'creator_language' => 'english'
+            'single_object_first_name' => 'Fake',
+            'single_object_last_name' => 'Fakerson',
+            'single_object_position' => 'Leader, Jester, Queen',
+            'single_object_language' => 'english'
           )
         end
 
         it 'succeeds' do
           metadata = subject.build_metadata
-          expect(metadata['creator']['first_name']).to eq('Fake')
-          expect(metadata['creator']['last_name']).to eq('Fakerson')
-          expect(metadata['creator']['position']).to include('Leader', 'Jester', 'Queen')
-          expect(metadata['creator']['language']).to eq('English')
+          expect(metadata['single_object']['first_name']).to eq('Fake')
+          expect(metadata['single_object']['last_name']).to eq('Fakerson')
+          expect(metadata['single_object']['position']).to include('Leader', 'Jester', 'Queen')
+          expect(metadata['single_object']['language']).to eq('English')
         end
       end
+
+      context 'with multiple objects and fields prefixed' do
+        let(:importer) do
+          FactoryBot.create(:bulkrax_importer_csv, field_mapping: {
+                              'multiple_objects_first_name' => { from: ['multiple_objects_first_name_1', 'multiple_objects_first_name_2'], object: 'multiple_objects' },
+                              'multiple_objects_last_name' => { from: ['multiple_objects_last_name_1', 'multiple_objects_last_name_2'], object: 'multiple_objects' },
+                              'multiple_objects_position' => { from: ['multiple_objects_position_1', 'multiple_objects_position_2'], object: 'multiple_objects' },
+                              'multiple_objects_language' => { from: ['multiple_objects_language_1'], object: 'multiple_objects', parsed: true }
+                            })
```

Comment:
Because we use this same "multiple objects" importer in our spec more than once, can we make it its own sub-factory or factory trait?